src/panels/lovelace/views/hui-view-background.ts

```diff
@@ -0,0 +1,130 @@
+import { css, LitElement, nothing } from "lit";
+import type { CSSResultGroup, PropertyValues } from "lit";
+import { customElement, property } from "lit/decorators";
+import type { HomeAssistant } from "../../../types";
+import type { LovelaceViewBackgroundConfig } from "../../../data/lovelace/config/view";
+
+@customElement("hui-view-background")
+export class HUIViewBackground extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) background?:
+    | string
+    | LovelaceViewBackgroundConfig
+    | undefined;
+
+  protected render() {
+    return nothing;
+  }
+
+  private _applyTheme() {
+    const computedStyles = getComputedStyle(this);
+    const themeBackground = computedStyles.getPropertyValue(
+      "--lovelace-background"
+    );
+
+    const fixedBackground = this._isFixedBackground(
+      this.background || themeBackground
+    );
+    const viewBackground = this._computeBackgroundProperty(this.background);
+    this.toggleAttribute("fixed-background", fixedBackground);
+    this.style.setProperty("--view-background", viewBackground);
+
+    const viewBackgroundOpacity = this._computeBackgroundOpacityProperty(
+      this.background
+    );
+    this.style.setProperty("--view-background-opacity", viewBackgroundOpacity);
```

Comment:
Why do we use variables for this? So it can also be used by themes? Then we should only set it if `_computeBackgroundOpacityProperty` doesn't return `null` right?

Reply:
The main use is to be able to use it in the css styling, like it was done for the background. For to be used in themes, there's a different theming variable used in the fallback: 

`var(--view-background,  var(--lovelace-background, var(--primary-background-color)));`